test/provision/runtime_install.sh

```diff
@@ -8,5 +8,12 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 source "${PROVISIONSRC}/helpers.bash"
 
+export GOPATH="/home/vagrant/go"
+sudo -E /usr/local/go/bin/go get -d github.com/lyft/protoc-gen-validate
+
+cd ${GOPATH}/src/github.com/lyft/protoc-gen-validate
+sudo git checkout 930a67cf7ba41b9d9436ad7a1be70a5d5ff6e1fc
```

Comment:
Could you please add comments to remind keeping this file in sync with the top-level Vagrantfile, and vice-versa?

Reply:
I would go a step further and put this hash in a variable in sourced file that is used by both this and to-level Vagrantfile.